site/content/articles/monitoring-multiple-openshift-clusters.adoc

```diff
@@ -0,0 +1,74 @@
+---
+title: "Monitoring Multiple Openshift Clusters"
+date: 2019-11-08T08:05:32+01:00
+authors: ["redhat-cop"]
+categories:
+  - openshift
+  - operations
+tags:
+  - research
+draft: true
+---
+= Monitoring Multiple Openshift Clusters
+Andrea Ferraresi <andrea.ferraresi@redhat.com>
+:toc: macro
+:toc-title:
+
+include::site/layouts/variables.adoc[]
+
+How to deal with monitoring of multiple Openshift Cluster: the challenges and the different approaches on scaling 
+
+toc::[]
+
+== Introduction to prometheus
+We all know Prometheus to be a very simple, very easy to use, piece of software:
+
+It is just a scraping server, this means that reads its metrics from a clear text page exposed by a web server on a specific port, metrics are generated from "exporters" which fills those pages. Once metrics are scraped are stored on disc for visualization. 
+
+It is not in the scope of Prometheus even to proper visualize those metrics, the UI is very simple and you would use Grafana for the graphing. 
+
+To keep things dead simple, there isn't any replication mechanism either.
+
+== The problems with prometheus
+
+This simplicity though comes with a cost: 
+
+*A*. Grafana can only query one Prometheus at a time which means that your monitoring server is a single point of failure: Having a single Prometheus instance per cluster is a single point of failure. 
```

Comment:
I think the colon is incorrect here, a full stop after failure is my recommendation.